src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -7,13 +7,12 @@
 namespace Revit.Elements.Views
 {
     /// <summary>
-    /// An abstract Revit View - All view types inherit from this type
+    ///     An abstract Revit View - All view types inherit from this type
     /// </summary>
-    //[SupressImportIntoVM]
```

Comment:
it was commented out, I just removed that line as per @ikeough earlier comments to not submit commented out code into production branch.